lib/version_utils.pm

```diff
@@ -195,38 +195,32 @@ sub check_version {
     croak "Unsupported version parameter for check_version: '$query'";
 }
 
-=head2 is_caasp
-
-Check if distribution is CaaSP or MicroOS with optional filter:
+=head2 is_microos
+Check if distribution is SUSE MicroOS or openSUSE MicroOS with optional filter:
 Media type: DVD (iso) or VMX (all disk images)
-Version: 1.0 | 2.0 | 2.0+
```

Comment:
Mixing `15.x` and `5.x` into the same comparison here might break some expectations, e.g. `is_microos("6.0+")` would also match `15.2`. At this point it's not clear how the version number would correspond (if at all), so I'd say this is something to fix when it becomes an issue.

Reply:
Yes, you are right and I plan to add that logic in [10968](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10968) for SUSE/Tumbleweed/Leap as arguments. So, let's fix it when needed.